vector/src/main/java/im/vector/app/core/pushers/VectorPushHandler.kt

```diff
@@ -78,27 +69,23 @@ class VectorMessagingReceiver : MessagingReceiver() {
     /**
      * Called when message is received.
      *
-     * @param context the Android context
-     * @param message the message
-     * @param instance connection, for multi-account
+     * @param pushData the data received in the push.
      */
-    override fun onMessage(context: Context, message: ByteArray, instance: String) {
-        Timber.tag(loggerTag.value).d("## onMessage() received")
+    fun handle(pushData: PushData) {
+        Timber.tag(loggerTag.value).d("## handling pushData")
 
-        val sMessage = String(message)
+        // tchap : use BuildCongig instead of BuildMeta
```

Comment:
I think this comment should be a TODO. It looks like a Tchap decision to keep BuildConfig, which is not true. This is just out of the scope here

"Tchap TODO: use BuildMeta instead of BuildConfig"